config/locales/cy.yml

```diff
@@ -153,7 +153,7 @@ cy:
           <li class="contact-panel__additional-info">Dydd Sadwrn, 9am i 1pm</li>
           <li class="contact-panel__additional-info">Dydd Sul a Gwyliau Banc, ar gau</li>
         </ul>
-      smallprint: Yr un gost â galwad arferol, os yw eich galwadau am ddim, caiff ei gynnwys.
+      smallprint: "Mae galwadau yn costio’r un faint â galw rhif 01 neu 02. Os oes gennych becyn galw “munudau am ddim”, gwiriwch gyda’ch darparwr bod rhifau 03 yn cael eu cynnwys."
```

Comment:
I believe the addition is consistent with what is already there but if there is a standard I'm happy to update to match that.